themes/src/main/resources/theme/base/email/messages/messages_en.properties

```diff
@@ -39,18 +39,11 @@ requiredAction.VERIFY_EMAIL=Verify Email
 requiredAction.CONFIGURE_RECOVERY_AUTHN_CODES=Generate Recovery Codes
 
 # units for link expiration timeout formatting
-linkExpirationFormatter.timePeriodUnit.seconds=seconds
-linkExpirationFormatter.timePeriodUnit.seconds.1=second
-linkExpirationFormatter.timePeriodUnit.minutes=minutes
-linkExpirationFormatter.timePeriodUnit.minutes.1=minute
-#for language which have more unit plural forms depending on the value (eg. Czech and other Slavic langs) you can override unit text for some other values like this: 
-#linkExpirationFormatter.timePeriodUnit.minutes.2=minuty
-#linkExpirationFormatter.timePeriodUnit.minutes.3=minuty
-#linkExpirationFormatter.timePeriodUnit.minutes.4=minuty
-linkExpirationFormatter.timePeriodUnit.hours=hours
-linkExpirationFormatter.timePeriodUnit.hours.1=hour
-linkExpirationFormatter.timePeriodUnit.days=days
-linkExpirationFormatter.timePeriodUnit.days.1=day
+linkExpirationFormatter.timePeriodUnit.seconds={0,choice,0#seconds|1#second|1<seconds}
+linkExpirationFormatter.timePeriodUnit.minutes={0,choice,0#minutes|1#minute|1<minutes}
+#for languages which have more unit plural forms depending on the value (eg. Czech and other Slavic langs) you can override unit text for some other values like described in the Java choice format: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/text/ChoiceFormat.html
```

Comment:
I'd link to https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/text/MessageFormat.html as well, or instead?

Reply:
Docs and link to docs are now updated.